Source/Projections/Builder/Copies/MongoDB/ConversionsFromBsonClassMapAdder.cs

```diff
@@ -0,0 +1,110 @@
+using System;
```

Comment:
The copyright header is missing, there are some other files with the same warning.